spec/index.bs

```diff
@@ -3568,7 +3568,13 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
             <div data-timeline=device>
                 [=Device timeline=] steps:
 
-                1. If |this| is [$invalid$], return.
+                1. If any of the following conditions are unsatisfied, return.
+
+                    <div class=validusage>
+                        - |this| is [$valid to use with$] |this|.{{GPUObjectBase/[[device]]}}.
```

Comment:
(This is good, it prevents trying to issue queue-timeline work if the device is lost.)

Reply:
Should we make this an assert as well? Losing the device unmaps all mapped buffers before it sets itself as invalid/lost.

Reply:
I don't think we can. Destroying the device unmaps all mapped buffers, but losing the device doesn't - device loss happens on the device timeline, so it can't unmap things out from under the content timeline. (It could do it by issuing a content-timeline task after it propagates to the content timeline, but we don't do that because it would still be sudden.)

Reply:
Right, that makes sense.